### PR TITLE
Fixes the problem introduced by the last PR request, cannot delete more than 1K objects 

### PR DESCRIPTION
**Pull request by rmoskal**
AWS deleteObjects chokes when given more than 1K files. I introduced this error with #176. I didn't try to delete thousands of files until yesterday.

Now chunking the input into this function.

Note, I used Promise.all to handle this, which meant I also had to change lintrc to allow E6 syntax.

Also I introduced a dependency on lodash.chunk.  I added a test for the chunking operation, but  did not feel writing a chunk function was the right thing to do.

I also updated the aws-sdk version (mostly in my initial desperation). Lemme know what you want to do about this.

### CODE REVIEW

**Comment by pioug** [comment]
> I also updated the aws-sdk version (mostly in my initial desperation). Lemme know what you want to do about this.

Ideally, I prefer when a PR changes only what's necessary to fix the bug. So when I look at previous commits (to revert changes, for curiosity...), I can understand more precisely what/why things are changed. But I don't want to be super fussy about it though 🙈 because your contribution is very welcomed 👍 

Anyway, since you did the fixed, I supposed you have something to test this case on hand? Maybe we can add that to the test suites?

File: package.json [hunk 1]
```diff
@@ -43,7 +45,6 @@
     "concurrent-transform": "^1.0.0",
     "coveralls": "*",
     "eslint": "^5.12.0",
-    "event-stream": "^4.0.1",
```
Comment (pioug) [inline package.json]: I think "event-stream" is not needed to run the plugin itself (only for the tests). Can we leave it in the "devDependencies"?

Comment (rmoskal) [reply package.json]: My error. I was having some difficulties with my local setup, npm link was doing something strange. I knocked back the version of aws-sdk to the one you have.

**Comment by pioug** [comment]
Did a squash-merge using the CLI https://github.com/pgherveou/gulp-awspublish/commit/59490e860518bc3732526b38160fc7977435710d. Thank you for the patch 👍  I'll push a new version soon.

**Pull request closed without merging**